package.json

```diff
@@ -201,7 +201,7 @@
     "async": "^3.2.4",
     "chai": "^4.1.2",
     "commander": "^7.0.0",
-    "eslint": "^7.32.0",
+    "eslint": "^8.24.0",
```

Comment:
this won't work as is because the eslint-config has peer dep versions in the 7.x range. I'll go ahead and fix that separately as it'll need to be done regardless of this so we can get on the latest eslint